src/App.js

```diff
@@ -1,24 +1,37 @@
-import logo from './logo.svg';
+/* eslint-disable import/no-extraneous-dependencies */
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import { Provider } from 'react-redux';
+// import store from './redux/configStore';
+import store from './store';
+import DashboardLayout from './layouts/dashboardLayout';
+import Login from './pages/Login';
+import Register from './pages/Register';
+// import User from './pages/User';
+import Home from './pages/home';
+import Booking from './components/booking';
+import Hotels from './components/hotels';
+import HotelShow from './components/hotelShow';
 import './App.css';
+import NotFound from './components/notfound';
 
 function App() {
```

Comment:
- [ ] When adding rooms, some fields of the form it is not possible to know what they are for.  Kindly add labels and/or placeholders so your users know what to include on each of those.  That way, the UX will be vastly improved. 👍 

![image](https://user-images.githubusercontent.com/57421823/215270970-f57cee47-2b24-4b7b-bf87-19e7f7f63738.png)